README.md

```diff
@@ -15,9 +15,19 @@ Credits go to [basmilius](https://github.com/basmilius) for the awesome [weather
 
 ## FAQ
 
-- [Why don't I see the current day in my weather forecast?](#why-dont-i-see-the-current-day-in-my-weather-forecast)
-- [Why does the forecast show less days than expected?](#why-does-the-forecast-show-less-days-than-expected)
-- [What does the card actually display?](#what-does-the-card-actually-display)
+- [Clock Weather Card](#clock-weather-card)
```

Comment:
was this change intentional? i don't want the full TOC here. only the TOC of FAQ

Reply:
It was accidentally auto generated by a VS Code extension.